src/frontend/ts/state/chessStore.ts

```diff
@@ -1,20 +1,40 @@
 import create from 'zustand'
 import Urbit from '@urbit/http-api'
 import { CHESS } from '../constants/chess'
-import { Update, Ship, GameID, GameInfo, ActiveGameInfo, Challenge, ChessUpdate, ChallengeUpdate, ChallengeSentUpdate, ChallengeReceivedUpdate, PositionUpdate, ResultUpdate, DrawOfferUpdate, DrawDeclinedUpdate, SpecialDrawPreferenceUpdate } from '../types/urbitChess'
+import { Update, Ship, GameID, SAN, FENPosition, GameInfo, ActiveGameInfo, Challenge, ChessUpdate, ChallengeUpdate, ChallengeSentUpdate, ChallengeReceivedUpdate, PositionUpdate, ResultUpdate, DrawOfferUpdate, DrawDeclinedUpdate, SpecialDrawPreferenceUpdate } from '../types/urbitChess'
 import { findFriends } from '../helpers/urbitChess'
 import ChessState from './chessState'
 
 const useChessStore = create<ChessState>((set, get) => ({
   urbit: null,
   displayGame: null,
+  displayMoves: [],
   practiceBoard: '',
   activeGames: new Map(),
   incomingChallenges: new Map(),
   outgoingChallenges: new Map(),
   friends: [],
+  reviewMode: false,
+  reviewIndex: null,
   setUrbit: (urbit: Urbit) => set({ urbit }),
-  setDisplayGame: (displayGame: ActiveGameInfo | null) => set({ displayGame }),
+  setDisplayGame: (displayGame: ActiveGameInfo | null) => {
+    if (displayGame !== null) {
+      set({ displayGame })
+      // XX: "GameInfo or ActiveGameInfo?" conditional logic
+      //
+      //     might be necessary depending on how we implement
+      //     the "view completed games" feature
+      get().setDisplayMoves([])
+      for (let move of displayGame.info.moves) {
+        get().displayMoves.push(move.san)
+      }
```

Comment:
Why not use `activeGameMoves` here?

Reply:
I think the thinking was:
1. We're setting `displayGame` here, so it feels more appropriate use `displayGame`'s list of moves.
2. It might be that not every `displayGame` being set here is an active game; they might be completed games. As gestured at in the comment here this function could change depending how #6 is implemented.

Reply:
I think we're too deep in the weeds here - I'll link what I'm getting at here shortly.

Reply:
We can get rid of `displayMoves` and `activeGameMoves` completely, since we're not using them. We'll have to account for this later, when dealing with archived games, but it won't be too bad since JS is garbage that allows hacky solutions to the hacky problems it creates. We can probably declare `displayGame` to be a union of `ActiveGameInfo` and `GameInfo` and just cast it depending on whether we're rendering an active or archived game.

Take a look at [this](https://github.com/ashelkovnykov/urbit-chess/commit/85e630c2e98152dcefd7615e28245463889a9c61) commit - it's what I was trying to get at with my past few comments about this specific block of code. It took 5 minutes to write and 2 hours to test (ty JS).

Reply:
Just poking @brbenji about the above comment.